traffic_ops/testing/api/v14/profiles_test.go

```diff
@@ -159,6 +161,69 @@ func GetTestProfiles(t *testing.T) {
 	}
 }
 
+func ImportProfile(t *testing.T) {
+	// Get ID of Profile to export
+	resp, _, err := TOSession.GetProfileByName(testData.Profiles[0].Name)
+	if err != nil {
+		t.Fatalf("cannot GET Profile by name: %v - %v\n", err, resp)
+	}
+	if resp == nil {
+		t.Fatalf("error getting Profile: response nil\n")
+	}
+	if len(resp) != 1 {
+		t.Fatalf("Profiles expected 1, actual %v\n", len(resp))
+	}
+	profileID := resp[0].ID
+
+	// Export Profile to import
+	exportResp, _, err := TOSession.ExportProfile(profileID)
+	if err != nil {
+		t.Fatalf("error exporting Profile: %v - %v\n", profileID, err)
+	}
+	if exportResp == nil {
+		t.Fatalf("error exporting Profile: response nil\n")
+	}
+
+	// Modify Profile and import
+
+	// Add parameter and change name
+	profile := exportResp.Profile
+	profile.Name = util.StrPtr("TestProfileImport")
+
+	newParam := tc.ProfileExportImportParameterNullable{
+		ConfigFile: util.StrPtr("config_file_import_test"),
+		Name:       util.StrPtr("param_import_test"),
+		Value:      util.StrPtr("import_test"),
+	}
+	parameters := append(exportResp.Parameters, newParam)
+	// Import Profile
+	importReq := tc.ProfileImportRequest{
+		Profile:    profile,
+		Parameters: parameters,
+	}
+	importResp, _, err := TOSession.ImportProfile(&importReq)
+	if err != nil {
+		t.Fatalf("error importing Profile: %v - %v\n", profileID, err)
```

Comment:
```gotest
    profiles_test.go:206: error importing Profile: 16 - 400 Bad Request[400] - Error requesting Traffic Ops https://localhost:6443/api/1.3/profiles/import {"alerts":[{"text":"validating: checking cdn name cdn1 existence, checking profile name TestProfileImport existence","level":"error"}]}
```
Not sure what's up with that, you definitely used CDNs in your WithObjs call.

Reply:
😕 this worked locally for me let me rerun. Thanks for testing it.